tests/lib/prepare-restore.sh

```diff
@@ -434,6 +437,13 @@ prepare_suite_each() {
 
 restore_suite_each() {
     rm -f "$RUNTIME_STATE_PATH/audit-stamp"
+
+    # restore test directory saved during prepare
+    if [ -f "${PWD}.tar" ]; then
+        rm -rf "$PWD"
+        tar -C/ -xf "${PWD}.tar"
+        rm -rf "${PWD}.tar"
```

Comment:
Wonder if we should `cd "$PWD"` at the end to avoid the weirdness of the directory being restored under a different inode, and current shell's $PWD being seemingly empty.

Reply:
Could be a follow-up.